.travis.yml

```diff
@@ -30,12 +28,12 @@ matrix:
     - php: 5.6
       env: DB=MYSQL BEHAT_TEST=1
     - php: 5.6
+      env: NPM_TEST=1
+  allow_failures:
```

Comment:
It's a temporary situation only (until I fix cms to also run in this way), and I don't actually think the allowed failures are running at the moment.